src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -4,49 +4,59 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.nullValue;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 
+import org.folio.circulation.support.http.client.IndividualResource;
+
+import api.support.APITestContext;
+import api.support.APITests;
+import api.support.http.OkapiHeaders;
+import io.vertx.core.json.JsonObject;
 import java.net.MalformedURLException;
 import java.util.UUID;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
-
-import org.folio.circulation.support.http.client.IndividualResource;
 import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
 import org.joda.time.DateTimeZone;
 import org.junit.Test;
 
-import api.support.APITestContext;
-import api.support.APITests;
-import api.support.http.OkapiHeaders;
-import io.vertx.core.json.JsonObject;
-
 public class ItemLastCheckInTests extends APITests {
 
+  @Override
+  public void beforeEach()
+    throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
+    super.beforeEach();
+    DateTimeUtils.setCurrentMillisFixed(1577457923195L);
```

Comment:
What does this set the time to?

I find this hard to interpret, can this use a date time rather than a long value?

Reply:
Sure.